src/Layout/index.js

```diff
@@ -70,3 +71,9 @@ const Bodybag = styled.div`
     will-change: scroll-position;
   `}
 `
+
+const ModalRoot = styled.div`
+  position: fixed;
+  z-index: 100000;
+  height: 100%;
```

Comment:
Can we remove `height: 100%` from here and move it to Tooltip? We can use portal for more than one type of modals in the future and not all of them may need 100%.